server/api/apps/shared_models/models/quiz_models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from . import defines
+from django.contrib.auth.models import User
```

Comment:
We are using a custom User model in the project. Please see this: https://docs.djangoproject.com/en/4.0/topics/auth/customizing/#referencing-the-user-model